packages/opencensus-exporter-object/src/object.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright 2019 OpenCensus Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Exporter, ExporterBuffer, ExporterConfig, Span} from '@opencensus/core';
+import {logger, Logger} from '@opencensus/core';
+
+/** Object Exporter manager class */
+export class ObjectTraceExporter implements Exporter {
+  buffer: ExporterBuffer;
```

Comment:
Can this be a private?

Reply:
To fix the build either make this variable public or remove check on buffer queue length.

Reply:
Build fixed.